packages/create-vite/src/index.ts

```diff
@@ -268,6 +268,19 @@ const FRAMEWORKS: Framework[] = [
       },
     ],
   },
+  {
+    name: 'marko',
+    display: 'Marko',
+    color: magenta,
+    variants: [
+      {
+        name: 'marko-run',
+        display: 'Marko Run ↗',
+        color: magenta,
+        customCommand: 'npm create marko',
```

Comment:
It is better to use `@latest` here to align with other custom commands, expect if you have a good reason to avoid it. npm will use the previously downloaded version in the user machine.
```suggestion
        customCommand: 'npm create marko@latest',
```
Given that we are first asking the user to specify a `TARGET_DIR`, not respecting it will be confusing. Is there a way to pass the target dir to the CLI command as we do for other custom commands? Would it be possible to add that feature to create-marko if not?

Reply:
Yep! Thanks, updated.